src/lib/tooltip/tooltip.ts

```diff
@@ -392,6 +427,113 @@ export class MdTooltip implements OnDestroy {
       this._tooltipInstance._markForCheck();
     }
   }
+
+
+  /**
+   * Registers the tooltip message for accessibility. If this message has not yet been
+   * registered, a new element will be created in a visually hidden container with the message
+   * as its content. This provides screenreaders a reference the tooltip trigger's aria-describedby.
+   * If an element has already been created for this message, increase that registered message's
+   * reference count.
+   */
+  private _registerA11yMessage() {
```

Comment:
What do you think about breaking all the code around this approach into a separate class called something like `AriaDescriber`? It would be good to capture all of this logic in one place separate from the tooltip and we could potentially re-use it in the future.